### DISCUSSION TITLE
Generally available : Azure functions in an isolated process .NET Framework 4.8

### DISCUSSION THREAD

**Discussion by cloudmelon** [Azure/azure-functions-dotnet-worker#957] [Announcements]
Apps can now target .NET Framework when running in the isolated process mode in Azure Functions v4. This allows apps with .NET Framework dependencies to explore the latest versions of Azure Functions. 

Apps built using this capability will follow the same patterns as any isolated .NET worker project in Functions, but they will specify .NET Framework 4.8 as the target framework. This is not intended to support production workloads, at this time, and some experiences may be limited. Please provide any feedback

Azure functions in isolated process .NET Framework 4.8 in Public Preview announcement : https://azure.microsoft.com/en-us/updates/public-preview-azure-functions-net-framework-support-in-the-isolated-worker-model/

GA Announcement : https://techcommunity.microsoft.com/t5/apps-on-azure-blog/announcing-general-availability-for-azure-functions-v4-net/ba-p/3637605

Please provide your feedback here : https://github.com/Azure/azure-functions-dotnet-worker/issues

**Comment by osiyash** [comment]
![image](https://user-images.githubusercontent.com/106608848/182312743-c8318c16-a120-4968-a5b5-33e72e3d5905.png)

can anyone help me , regarding this error

**Reply by cloudmelon** [reply]
Hi @osiyash  This issue looks like from CosmosDB side, do you have mocked data ?
